code/go/0chain.net/smartcontract/dbs/event/process.go

```diff
@@ -62,6 +62,7 @@ func (edb *EventDb) ProcessEvents(
 	for i := range es {
 		localCounter++
 		es[i].SequenceNumber = int64(latestGlobalCounter) + int64(localCounter)
+		es[i].RoundLocalSequenceNumber = int64(localCounter)
```

Comment:
what is the key of a message, is it synchronized across sharders?

Reply:
added this